com/yadro/HWManager/Fan.interface.yaml

```diff
@@ -45,3 +45,9 @@ properties:
         - readonly
       description: >
           Index of Fan PWM line
+    - name: speed_limit_max
```

Comment:
Looks like this needs to be called `pwm_limit_max`. Speed is in RPM.

Reply:
> Looks like this needs to be called `pwm_limit_max`. Speed is in RPM.

It is in percent